services/galley/src/Galley/API/Teams.hs

```diff
@@ -336,6 +337,29 @@ updateTeamH zusr zcon tid updateData = do
   let r = list1 (userRecipient zusr) (membersToRecipients (Just zusr) (memList ^. teamMembers))
   E.push1 $ newPushLocal1 (memList ^. teamMemberListType) zusr (TeamEvent e) r & pushConn ?~ zcon
 
+updateTeamDeleteSplashScreenH ::
+  Members
+    '[ ErrorS 'NotATeamMember,
+       ErrorS ('MissingPermission ('Just 'SetTeamData)),
+       GundeckAccess,
+       Input UTCTime,
+       TeamStore
+     ]
+    r =>
+  UserId ->
+  ConnId ->
+  TeamId ->
+  Sem r ()
+updateTeamDeleteSplashScreenH zusr zcon tid = do
+  zusrMembership <- E.getTeamMember tid zusr
+  void $ permissionCheckS SSetTeamData zusrMembership
+  E.deleteTeamSplashScreen tid
+  now <- input
+  memList <- getTeamMembersForFanout tid
+  let e = newEvent tid now (EdTeamUpdate newTeamUpdateData)
```

Comment:
I'm not very happy about this: deleting the splash screen is not the same as changing nothing.  :)  Maybe add a new constructor to `TeamUpdateData`?